src/middlewared/middlewared/plugins/gluster_linux/peer.py

```diff
@@ -38,6 +44,32 @@ def __peer_wrapper(self, method, host=None):
 
         return result
 
+    def _parse_peer(self, p):
+
+        data = {
+            'uuid': p.find('uuid').text,
+            'hostname': p.find('hostname').text,
+            'connected': p.find('connected').text,
+        }
+
+        if data['connected'] == '1':
+            data['connected'] = 'Connected'
```

Comment:
Why do you use string typing for a boolean value?

Reply:
The subprocessed command that gets run returns XML data so it gets returned as a string.